cmd/podman/system/connection/list.go

```diff
@@ -84,6 +103,9 @@ func list(_ *cobra.Command, _ []string) error {
 	}
 	defer w.Flush()
 
-	_ = tmpl.Execute(w, hdrs)
+	isTable := report.HasTable(cmd.Flag("format").Value.String())
```

Comment:
This works but template seems to have a .isTable boolean value that is not set when New() is called. within [Parse()](https://github.com/containers/common/blob/9bc6b6ff91d73c5c6fc2d784252b310eb820eb76/pkg/report/template.go#L130 ) isTable is set and even returned but inaccessible?

Reply:
NormalizeFormat() strips table from the format, making IsTable() false.